ModConfigMenu/ModConfigMenu/Src/ModConfigMenuAPI/Classes/MCM_API_Instance.uc

```diff
@@ -1,5 +1,13 @@
 interface MCM_API_Instance;
 
+enum eSettingEventMode
+{
+    // Callbacks are triggered when the user manipulates the UI element. Use SetSaveHandler and SetCancelHandler to save/abort.
+    eSettingEventMode_Immediate,

```

Comment:
I _really_ don't like that this setting as an option.  It's going to make the behavior of the API more inconsistent and bug-prone, for both us and users of the API.

Instead, as suggested in #13, the `OnChanged` callbacks should be called when the values are changed, but that's _not_ where the config variables should be set.  Instead, they should be set only in `OnSave`